src/coredata.cs

```diff
@@ -2677,6 +2725,43 @@ interface NSPersistentCloudKitContainer {
 		[Watch (7,0), TV (14,0), Mac (11,0), iOS (14,0)]
 		[Export ("canModifyManagedObjectsInStore:")]
 		bool CanModifyManagedObjects (NSPersistentStore store);
+
+		// NSPersistentCloudKitContainer_Sharing
+		[Async (ResultTypeName = "NSPersistentCloudKitContainerAcceptShareInvitationsResult")]
+		[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
+		[Export ("acceptShareInvitationsFromMetadata:intoPersistentStore:completion:")]
+		void AcceptShareInvitations (CKShareMetadata[] metadata, NSPersistentStore persistentStore, [NullAllowed] NSPersistentCloudKitContainerAcceptShareInvitationsHandler handler);
```

Comment:
the `NSPersistentCloudKitContainerAcceptShareInvitationsHandler` has `[NoWatch]` and `[NoTV]` attributes